etna/search/tests/test_views.py

```diff
@@ -1717,3 +1717,47 @@ def test_record_creators_country_long_filter(self):
                 "&size=20"
             ),
         )
+
+
+class CreateSessionInfoForBackToSearchTest(SearchViewTestCase):
+    @responses.activate
+    def test_create_session_info_for_catalogue_search(self):
+        search_url = reverse("search-catalogue")
+        expected_url = "/search/catalogue/?sort_by=title&q=london&filter_keyword=paper&level=Item&collection=ADM&collection=BT&closure=Open+Document%2C+Open+Description&opening_start_date_0=&opening_start_date_1=&opening_start_date_2=1900&opening_end_date_0=&opening_end_date_1=&opening_end_date_2=2020&per_page=20&sort_order=asc&display=list&page=2&group=tna"
```

Comment:
These are pretty heavy-weight tests for just checking that a session value is set. Could we see instead about extending one of the existing tests to include the check in a single request/response cycle?